### ISSUE TITLE
React App template

### ISSUE THREAD

**Issue by ok2ju** [jszombies/create-zombie-lib#3]
Add react app template (like https://github.com/ok2ju/modern-react-boilerplate)

**Comment by alehatsman** [comment]
Yes for sure. Just let's define what is the purpose of the template and what should be included.
Last days I was working on nextjs app. I created that template https://github.com/jszombies/create-zombie-lib/tree/master/templates/react-nextjs-app

**Comment by ok2ju** [comment]
It's boilerplate just for regular react application.
I think what it should have the following:
- standard for linting (lets try it out);
- webpack for bundling;
- jest + enzyme for tests;
- css modules (because its most common and simple approach now);
- redux;
- redux-saga;
- react-router (im not sure about version, I only used v3).

**Comment by shamin** [comment]
@ok2ju @Atsman  I have a simple react starter boilerplate with eslint for linting and sass support available at https://github.com/shaminmeerankutty/react-starter. I can also add jest, enzyme , redux, react-router (v4) to starter. Is it ok?

**Comment by ok2ju** [comment]
@shaminmeerankutty it looks pretty good and simple, I like it, but we came to decision that we need tools like I described above except of `css modules` (we should replace it with `styled-components`).

`standard` instead of `eslint`
`styled-components` instead of `css/sass`

If you agree with this requirements we will be happy to add your starter into our CLI. We appreciate any contribution 😃 

@Atsman what do you think?
